CHANGES.rst

```diff
@@ -448,6 +448,9 @@ Bug fixes
   - Fix a problem where the fast reader (with use_fast_converter=False) can
     fail on non-US locales. [#4363]
 
+  - Fix astropy.io.ascii.read handling of units for IPAC formatted files.
+    Columns with no unit are treated as unitless not dimensionless.
```

Comment:
Should include this PR number in your change log entry (see other entries for example).